src/nunavut/lang/cpp/support/serialization.j2

```diff
@@ -14,7 +14,7 @@ namespace nunavut
 {
 namespace support
 {
-
+// Target Endianess: {{ option_target_endianness }}
```

Comment:
Where does this option come from exactly? I am asking because I intended to verify that the available options are "little" and "any" (rather than "little" and "big").

Reply:
Added 'any' as a valid option. It's up to the template author to consume these values, ultimately. If I'm wrong I'll eat my hat but I will not, under any circumstances, eat my dog.

Reply:
Another useful phrase to have in your Russian lexicon is "[let there be] a dog with (subject)" ("пёс с ним"), which conveys indifference towards the subject.